packages/ui/src/components/wui-text/styles.ts

```diff
@@ -130,4 +130,8 @@ export default css`
   .wui-font-micro-600 {
     font-weight: var(--wui-font-weight-medium);
   }
+
+  :host([disabled]) {
+    opacity: 0.5;
```

Comment:
This should be 0.4 for the text, mentioned [here](https://walletconnect.slack.com/archives/C06B51DMENL/p1721739748814159?thread_ts=1721375493.320809&cid=C06B51DMENL)